exercises/collatz-conjecture/test/collatz_conjecture_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:test/test.dart';
+import 'package:collatz_conjecture/collatz_conjecture.dart';
```

Comment:
The imports need to be sorted alphabetically :)

Reply:
Ah, yes. Sorry, I always forget about that. I'll see if I can get a fix out soon, if not sometime tonight.